packages/test-utils/src/wrapper-array.js

```diff
@@ -23,7 +23,8 @@ export default class WrapperArray implements BaseWrapper {
   }
 
   at(index: number): Wrapper | VueWrapper {
-    if (index > this.length - 1) {
+    index = index < 0 ? this.length + index : index
```

Comment:
should we

```suggestion
    const normalizedIndex = index < 0 ? this.length + index : index
```

Otherwise for an array of length 1, the error thrown for `wrapper.at(-2)` will be `no item exists at -1` which could cause confusion.

Reply:
Yeah I wasn't sure about that either, but at the same it's useful to see the computed index in the error message. So I changed the message when the index is negative, so can see both the original argument and the computed one. What do you think @winniehell?

Reply:
looks good—thank you! :tada: